src/Compilers/CSharp/Test/Syntax/LexicalAndXml/LexicalTests.cs

```diff
@@ -4577,15 +4577,20 @@ disabled text 2
         [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/78593")]
         public void TestDotPrefixedNumberStartingAtStartOfSlidingTextWindow()
```

Comment:
even though the window is now handle very differently, i still kept this test, but augmented it.  it's still worth validating that if we need to peek back one char, and taht reads in a different chunk of text from the source-text, that that works properly.